mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -73,9 +73,8 @@ def __remote_init(self):
 
         # Remote DUT connection, flashing and reset...
         try:
-            self.__remote_connect(baudrate=self.baudrate)
-            self.__remote_flashing(self.image_path)
             self.__remote_disconnect()
```

Comment:
Yea, it seems to have a check for this. lgtm, though I have no experience with htrun so I'll leave it for someone else to approve this.